pages/linux/e2fsck.md

```diff
@@ -1,16 +1,20 @@
 # e2fsck
 
-> Check a Linux ext2/ext3/ext4 filesystem. The filesystem should be unmounted at the time the command is run.
```

Comment:
I have a minor preference that we keep the original text; passive voice is more appropriate when the actor is unknown and we want to emphasize the action itself.

That said that's a non-blocker and we can move ahead with this if someone else signs off.

Reply:
I would still prefer "Unmount the partition first!", but you require a dot. The original is too long either way, and unnecessarily complicated in structure.